### PR TITLE
Emails refresh - updating text in email templates

### PR DESCRIPTION
**Pull request by alanna**
Updated the content of all email templates under /templates/emails. Removed outdated info, fixed grammar, simplified messaging and elements of styling where possible. 

Did not update hardcoded custom thankyou templates because I'd need to reach out to those particular users.

Related to https://github.com/opencollective/opencollective-frontend/pull/6135

### CODE REVIEW

File: templates/emails/collective.contact.hbs
Comment (SudharakaP) [inline templates/emails/collective.contact.hbs:29]:
```suggestion
<p>If this message is spam, please forward it to <a href="mailto:support@opencollective.com">support@opencollective.com</a>.</p>
```

File: templates/emails/collective.created.the-social-change-nest.hbs
Comment (SudharakaP) [inline templates/emails/collective.created.the-social-change-nest.hbs:15]:
```suggestion
<p>After filling in the form, the next step is to book a <b>45 minute on-boarding call</b> with us, which you can do <a href="https://calendly.com/accountableonboarding">here</a>. If none of these times work for you, <a href="mailto:accountable@thesocialchangeagency.org">get in touch</a>.</p>
```

File: templates/emails/collective.expense.error.for.host.hbs
Comment (SudharakaP) [inline templates/emails/collective.expense.error.for.host.hbs:30]:
```suggestion
    Check TransferWise to find out what happened. You might need to send this payment manually through TransferWise or your bank, then mark it as paid on Open Collective.<br />
```

File: templates/emails/user.card.claimed.text.hbs
Comment (SudharakaP) [inline templates/emails/user.card.claimed.text.hbs:1]:
```suggestion
Click on the link below to login to your account and support the Collectives you love.
```

File: templates/emails/collective.expense.error.for.host.hbs
Comment (alanna) [reply templates/emails/collective.expense.error.for.host.hbs:30]: Transferwise change its name to just Wise.

Comment (SudharakaP) [reply templates/emails/collective.expense.error.for.host.hbs:30]: @alanna : Ah, didn't know about that. Thought it was a typo. 👍🏽

File: templates/emails/collective.expense.error.for.host.hbs
Comment (SudharakaP) [inline templates/emails/collective.expense.error.for.host.hbs:31]:
```suggestion
    If you think the problem will not reoccur, you can also retry clicking the "Pay with Wise" button.
```

**Comment by SudharakaP** [comment]
Just to note, this should be merged with, https://github.com/opencollective/opencollective-frontend/pull/6135

**Review by SudharakaP — APPROVED** [review]

**Comment by alanna** [comment]
@kewitz @Betree can this be merged even though @znarf is on holiday? Would be great to get it out there.

@SudharakaP thanks very much for your review!

**Comment by Betree** [comment]
> @kewitz @Betree can this be merged even though @znarf is on holiday? Would be great to get it out there.
> 
> @SudharakaP thanks very much for your review!

It's surely can! Thanks to @SudharakaP's PR to fix the tests in the frontend everything is green so it's just a matter of checking the files to make sure we didn't miss anything.

I'll give a proper review tomorrow and will merge if there's no blocker.

**Comment by SudharakaP** [comment]
> @SudharakaP thanks very much for your review!

You are welcome. 😄

**Review by Betree — APPROVED** [review]
Wow this was a massive update :smile: That looks great, nice job @alanna! I made some suggestions below that I'll apply before merging. I've also added [`fbe9411`](https://github.com/opencollective/opencollective-api/pull/5801/commits/fbe9411ae2baad0756dc5a10cb0206875c40f484) to remove `topBackers` query (was not used anymore) and fix updates info in monthly report (was broken before this PR)

**Pull request merged**
